modules/installation-user-infra-machines-iso.adoc

```diff
@@ -17,33 +17,30 @@ the machines that you create can access.
 
 .Procedure
 
-////
-This will work for GA.
-. Obtain the {op-system} images from the
-link:https://access.redhat.com/downloads[Product Downloads] page on the Red
-Hat customer portal.
-////
-
 . Upload the control plane, compute, and bootstrap Ignition config files that the
 installation program created to your HTTP server. Note the URLs of these files.
 
-
 . Obtain the {op-system} images that are required for your preferred method
 of installing operating system instances from the
 link:https://access.redhat.com/downloads[Product Downloads] page on the Red
 Hat customer portal or the
 link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.1/[{op-system} image mirror]
 page.
 +
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the highest version that is less than or equal
+to the {product-title} version that you install. Use the image versions
+that match your {product-title} version if they are available.
+====
++
 You must download the ISO file and either the BIOS or UEFI file.
 Those file names resemble the following examples:
 
-** ISO: `rhcos-<version>-<date>-installer.iso`
-** Compressed metal BIOS: `rhcos-<version>-<date>-metal-bios.raw.gz`
-** Compressed metal UEFI: `rhcos-<version>-<date>-metal-uefi.raw.gz`
-
-. Obtain the {op-system} images that are required for your preferred method
-of installing operating system instances.
+** ISO: `rhcos-<version>-installer.iso`
+** Compressed metal BIOS: `rhcos-<version>-metal-bios.raw.gz`
+** Compressed metal UEFI: `rhcos-<version>-metal-uefi.raw.gz`
```

Comment:
It's technically `rhcos-<version>-<architecture>-metal-uefi.raw.gz` (e.g. `rhcos-4.1.0-x86_64-metal-uefi.raw.gz`) and similar. This should be obvious to anyone who goes and looks at what's there, so I'm not really concerned about it if you want to roll in version+arch into `<version>`, just wanted to make sure you noticed that.

Reply:
I fixed it. If we eventually add Power or z support, it'll be an important designation.